changelog/fragments/operator-name-config.yaml

```diff
@@ -0,0 +1,14 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Remove `--operator-name` from `generate` subcommands in favor of using `project-name`
+      from the PROJECT config file (v3-alpha+).
+    kind: removal
+    breaking: true
+    migration:
+      header: Remove `--operator-name` from scripts
+      body: >
+        The `--operator-name` flag has been removed from `generate bundle` and `generate packagemanifests`
+        subcommands. Remove this flag from your scripts, and make sure the `project-name` key is set in
+        your PROJECT file. If this key is not set, the current working directory's base name will be used.
```

Comment:
Master branch creating the PROJECT with the projectName yet. So should we not need to merge this first? 

```
 $ cat PROJECT 
domain: my.domain
layout: go.kubebuilder.io/v2
repo: example
version: 3-alpha
plugins:
  go.operator-sdk.io/v2-alpha: {}

```

Reply:
Right, this still needs to be changed for the v2 plugin only if project version is v3+.

Reply:
https://github.com/kubernetes-sigs/kubebuilder/pull/1611